src/blocks/ImageCard.tsx

```diff
@@ -174,6 +156,24 @@ const ImageCard = (props: ImageCardProps) => {
           )}
         </figure>
         {getStatuses()}
+        <div className="image-card-tag__wrapper">
```

Comment:
One thing that this change does is it makes the screen reader read the tags last. So for this example the "application due date" is read before the "Seniors 62+". 
![image](https://user-images.githubusercontent.com/42942267/220147252-5478160e-c74d-4dc6-b30a-3da884fc0de6.png)

I don't think this is an issue but @ColinBuyck & @emilyjablonski do you have any opinions on this?

Reply:
That seems probably preferable, I'll link to core and test out as well

Reply:
Thank you for calling that out @ludtkemorgan!